content/en/docs/tasks/security/authz-http/index.md

```diff
@@ -32,178 +30,198 @@ the product page, you can see the following sections:
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: security.istio.io/v1beta1
+    kind: AuthorizationPolicy
+    metadata:
+      name: deny-all
+      namespace: default
+    spec:
+      {}
+    EOF
+    {{< /text >}}
+
+    Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`).
+    You should see `"RBAC: access denied"`. The error shows that the configured `deny-all` policy
+    is working as intended, and Istio doesn't have any rules that allow any access to
+    workloads in the mesh.
+
+    With Istio, you can easily configure namespace-level access control. You can configure how could workloads
+    from a namespace access workloads in another namespace.
+
+1. Run the following command to create a `bookinfo-viewer` policy that allows workloads in the `default`
+   and `istio-system` namespaces to access workloads in the `default` namespace with `GET` method:
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: "security.istio.io/v1beta1"
+    kind: "AuthorizationPolicy"
+    metadata:
+      name: "bookinfo-viewer"
+      namespace: default
+    spec:
+      rules:
+      - from:
+        - source:
+            namespaces: ["default", "istio-system"]
+        to:
+        - operation:
+            methods: ["GET"]
+    EOF
+    {{< /text >}}
+
+    Point your browser at Bookinfo's `productpage` (`http://$GATEWAY_URL/productpage`).
+    You should see the "Bookinfo Sample" page, with the "Book Details" section in the lower left part
+    and the "Book Reviews" section in the lower right part.
+
+1. Remove the policy with the following command before continuing:
+
+    {{< text bash >}}
+    $ kubectl delete authorizationpolicy.security.istio.io/bookinfo-viewer
+    {{< /text >}}
+
+1. Run the following command to create a `productpage-viewer` policy which allows
+   access to the `productpage` workload with `GET` method for all users and workloads:
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: "security.istio.io/v1beta1"
+    kind: "AuthorizationPolicy"
+    metadata:
+      name: "productpage-viewer"
+      namespace: default
+    spec:
+      selector:
+        matchLabels:
+          app: productpage
+      rules:
+      - to:
+        - operation:
+            methods: ["GET"]
+    EOF
+    {{< /text >}}
+
+    Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`).
+    Now you should see the "Bookinfo Sample" page. But there are errors `Error fetching product details`
+    and `Error fetching product reviews` on the page.
+
+    These errors are expected because we have not granted the `productpage`
+    workload access to the `details` and `reviews` workloads. We will fix the
+    errors in the following steps.
+
+1. Run the following command to create a policy `details-viewer` to allow the `productpage`
+   workload (represented by `principals: ["cluster.local/ns/default/sa/bookinfo-productpage"]`)
```

Comment:
`represented by` might be well known jargon within the security team but not for our users. Please use simpler wording.
```suggestion
   workload, which issues requests using the `cluster.local/ns/default/sa/bookinfo-productpage` service account,
```